app/views/widgets/taskstatus.html

```diff
@@ -0,0 +1,14 @@
+{% macro taskStatus(content, title, status, previousTaskStatus) %}
```

Comment:
This macro is handling the title as well as the status so perhaps it would be more accurate to call it something like `taskHeader` rather than `taskStatus`?